src/Azure/Orleans.Persistence.AzureStorage/Providers/Storage/AzureTableStorageOptions.cs

```diff
@@ -19,6 +19,11 @@ public class AzureTableStorageOptions : AzureStorageOperationOptions, IStoragePr
         /// </summary>
         public bool DeleteStateOnClear { get; set; } = false;
 
+        /// <summary>
+        /// Indicates if grain data should be stored as a json string or in binary format.
+        /// </summary>
+        public bool UseJson { get; set; }
```

Comment:
I think a better name would be `UseStringFormat", no? I don't think it's limited to json

Reply:
Just wanted to conform the old config, but I agree with you